docs/usage.rst

```diff
@@ -1,62 +1,78 @@
 Using ``pyramid_nacl_session``
 ==============================
 
-Generating a Random Secret
---------------------------
+Setup
+-----
 
-First, you need to arrange to have a 32-byte random secret available:
+Once :mod:`pyramid_nacl_session` is installed, you typically use the ``config.include`` mechanism to include it into your Pyramid project's configuration.
```

Comment:
Here and below, please wrap at 80 characters.

Reply:
This request for change is inconsistent with https://docs.pylonsproject.org/projects/docs-style-guide/#use-of-whitespace

If we cannot agree on a standard as an organization, then we should revise this section in the guide, giving a consistent message to contributors.